samples/test/upload-model.test.js

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
(for the jpg images above, sorry there's no way to leave comments there) - I generally try to avoid binaries in my git repositories if I can.  There's no way for git to do it's diffing magic, so this bloats the index size significantly, especially as the contents change over time.  Would it make sense to keep this data in one of our 4 GCS buckets we're using, and then have the tests download the file before using it?

Reply:
Y'know what, that's an oversight on my part. Those images are a relic of some samples that demonstrated using local resources. I removed those samples. We don't even need them.